src/main.rs

```diff
@@ -345,12 +347,12 @@ fn main() -> BError {
             default_stats.clone(),
         );
         // Creep spawners
-        centity!(
-            world,
-            Point::new(x, y - 1),
-            CreepSpawner(0, CREEP_SPAWN_TICKS),
-            Team::Me,
-        );
+        // centity!(
```

Comment:
Why is this creep spawner disabled?

Reply:
Oops, I was testing and forgot to uncomment. I amend the fix!